kubeflow/common/iap-ingress/base/policy.yaml

```diff
@@ -8,7 +8,7 @@ spec:
       app: istio-ingressgateway
   jwtRules:
   - audiences:
-    - TO_BE_PATCHED
+    - JWT_AUDIENCE
```

Comment:
@zijianjoy , let me know if you have a better name in mind.

Reply:
This name sounds good to me, thanks!